recipes/temporal-cache/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "temporal-cache" %}
+{% set version = "0.0.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 31cccf3830497336884ae1748a60011e723cd6d312789b6c049bec6ddb3e0f89
+
+build:
+  number: 0
+  skip: True  # [py<37]
```

Comment:
I think this recipe can be `noarch`, so please remove the `skip` line, and add `noarch: python` instead.